fsharp-backend/src/Wasm/Wasm.fs

```diff
@@ -260,51 +264,50 @@ module Eval =
         (List.map ClientInterop.convert_db af.dbs)
         af.func.ast
 
-
-
 open System
 open System.Reflection
 
 #nowarn "988"
 
 type GetGlobalObjectDelegate = delegate of string -> obj
 
-// This is F# equivalant of the C# delegate:
-// public delegate object InvokeDelegate(string m, params object[] ps);
 type InvokeDelegate = delegate of m : string * [<ParamArray>] ps : obj [] -> obj
 
+/// Responsible for interacting with the JS world
+///
+/// Exposes fns to be called to evaluate expressions,
+/// and results back to JS
 type EvalWorker =
-  // Create a delegate with which to call self.postMessage
-  static member getPostMessageDelegate() : InvokeDelegate =
-    let assemblyName = "System.Private.Runtime.InteropServices.JavaScript"
-    let typeName = "System.Runtime.InteropServices.JavaScript.Runtime"
-
-    // Get the `self` object from the webworker.
-    let sourceAssembly : Assembly = Assembly.Load(assemblyName)
-    let typ = sourceAssembly.GetType(typeName)
+  static member GetGlobalObject(_globalObjectName : string) : unit = ()
+
+  static member selfDelegate =
```

Comment:
Are `static member x()` and `static member x` equivalent? If not, why the change?

Reply:
They're not the same - the first of these requires the unit param.
In this case, the unit param seemed useless, and I have a habit of removing useless params (if something can be just a simple value, why not?)

In this specific case, by the end of executing this file `postMessageDelegate` was already resolved (no hanging () argument) anyway, so it feels the same to me. `getPostMessageDelegate()` had a `()` param, but was only used to create `postMessageDelegate`. This all felt like more noise than useful, so I simplified to a plain `selfMessageDelegate`

Reply:
I meant given it's called the same way before and after, did it work before or does it work now or if both then why?

Not a big deal, if it works it works, I'm just confused.